### DISCUSSION TITLE
Overwrite Form Values and Save With getValues()

### DISCUSSION THREAD

**Discussion by jostal** [react-hook-form/react-hook-form#7954] [Q&A]
So, I have a form and I save the values to a db by using getValues(). However, in the form there is a search feature which returns some information in a table. I need to save this table info. In order to use getValues(), I decided to make hidden inputs that hold the values I need to save. 
The problem is, these values are displayed conditionally and can change from being shown to not shown and when they are not shown, I need to set these values to null. I tried to do this by creating the hidden inputs which display the values conditionally, and it appears to work if I console.log the fieldArray and if I inspect element, the inputs that should not exist do not, so it appears all good. Despite this, getValues() is still returning whatever the original values were, and not what the current ones appear to be. 
How can I overwrite these values? Alternatively, is there a better way to use getValues() and save the table info?
Thanks in advance.

**Comment by bluebill1049** [comment]
>getValues() is still returning whatever the original values were, and not what the current ones appear to be. 

can you share a simple codesandbox?
